### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by seanye24**
Add list of project dependencies in `requirements.txt`.

### CODE REVIEW

**Review by jmazanec15 — APPROVED** [review]

**Comment by VijayanB** [comment]
We should use pip compile instead of generating requirements.txt manually.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Cerberus==1.3.4
+certifi==2021.5.30
+elasticsearch==7.13.3
```
Comment (VijayanB) [inline requirements.txt:3]: @jmazanec15 may be 7.10.2?

Comment (VijayanB) [reply requirements.txt:3]: Do we really need dependency to elasticsearch library here?

Comment (jmazanec15) [reply requirements.txt:3]: Yeah 7.10.2 makes sense. We need it in order to communicate with the OpenSearch cluster. I think there may be an OpenSearch python client we can use, but I am not sure it is ready: https://discuss.opendistrocommunity.dev/t/opensearch-client-java-python-apis/5639/20. Until then, the elasticsearch python client should work and it has AL2 license.

**Comment by jmazanec15** [comment]
> We should use pip compile instead of generating requirements.txt manually.

How does this work? Do you have link?

File: requirements.txt (hunk 1 above)
Comment (VijayanB) [reply requirements.txt:3]: Why not use request library ?

**Comment by VijayanB** [comment]
> > We should use pip compile instead of generating requirements.txt manually.
> 
> How does this work? Do you have link?

https://github.com/jazzband/pip-tools

**Comment by seanye24** [comment]
> We should use pip compile instead of generating requirements.txt manually.

I've updated the `requirements.txt` using `pip-compile`.

**Comment by seanye24** [comment]
Moving this PR to #3.

**Pull request closed without merging**